### PR TITLE
OPS: Some environments required this variable to build

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/o3l4lemdanhm2yfi7voimrrz4i?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/11020)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

File: android/build.gradle [hunk 1]
```diff
@@ -15,6 +15,8 @@ buildscript {
         // We use NDK 23 which has both M1 support and is the side-by-side NDK version from AGP.
         ndkVersion = "23.1.7779620"
         kotlin_version = '1.8.0'
+        kotlinVersion = '1.8.0'
```
Comment (Overtorment) [inline android/build.gradle:18]: weird

Comment (Overtorment) [reply android/build.gradle:18]: which environments? pls elaborate

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

File: android/build.gradle (hunk 1 above)
Comment (marcosrdz) [reply android/build.gradle:18]: On my m2 mac. Rn tor and rnldk failed due to that missing variable.

**Pull request merged**